### ISSUE TITLE
Fresh demo deployment nightly

### ISSUE THREAD

**Issue by jgrim** [sublinks/sublinks-api#61]
Each night, when the demo site is updated with the nightly build, it should also refresh to a specific database state.

My suggestion is to set up the demo site and define the requested details: languages, Site sidebar, demo communities, demo posts, comments, etc. Create an image of that container. Use that image as the base image with every deployment to the server.

After the nightly build image is updated, running the command `docker compose down && docker compose up -d` should update the sublinks container to the latest imported snapshot while recreating all the other containers: db, media, etc.

You can implement it however you wish—just a thought.

tl;dr; Make it so the demo site refreshes consistently every night, wiping any changes from the day away.
